src/components/ImageAsset.vue

```diff
@@ -129,6 +134,10 @@ export default {
       type: Array,
       required: true,
     },
+    loading: {
+      type: String,
+      default: 'lazy'
```

Comment:
```suggestion
      default: 'lazy',
```

(Super tiny thing, but the linter is complaining about this)